Header header
string cam_type
string image3D
string image_surr
string cam_motion
